index.html

```diff
@@ -78,6 +78,165 @@ <h1 class="heading">
           </div>
         </div>
       </section>
+
+      <!-- Works/projects -->
+      <section class="grid-container">
+        <h1 class="heading-container heading">My Recent Works</h1>
+        <br />
+        <!-- Line -->
+        <div class="line"></div>
+
+        <!-- Boxes -->
+        <div class="boxes">
+          <div class="project-tile">
+            <div class="box-1 project-tile">
+              <h1 class="tile-heading">
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h1>
```

Comment:
- [ ] I would suggest you to change the `h1 tag` and use `hx` like `h2`, `h3`, and so on because per page, you are allowed to use only one `h1 tag` for better semantic HTML.